src/schema_refined/core.clj

```diff
@@ -43,6 +43,30 @@
 (defn predicate? [p]
   (satisfies? Predicate p))
 
+(defn schema->str [schema]
+  {:pre [(schema? schema)]}
+  (let [tag (:schema-refined.core/tag (meta schema))]
```

Comment:
Why not use `::tag`?

Reply:
Old habits, easy to mess up when refactoring & splitting code to a different namespaces